BitwardenShared/Core/Tools/Repositories/ExportCXFCiphersRepository.swift

```diff
@@ -0,0 +1,129 @@
+import AuthenticationServices
+import BitwardenSdk
+
+/// Protocol for a repository to handle exporting ciphers in Credential Exchange Format.
```

Comment:
⛏ could you add extra line
```suggestion
/// Protocol for a repository to handle exporting ciphers in Credential Exchange Format.
///
```

Reply:
Oh I've seen that extra line in some places, what's the reason behind adding that? Does it show a better format when compiling the docs or something like that? Or what's the advantage of it?

Reply:
We're extremely inconsistent about applying it throughout the codebase, as well. It might be good for us to put something like that in a style guide somewhere if that's what we actually want?